### PR TITLE
Update of jet config and GNU regression tests

### PR DESCRIPTION
**Pull request by climbfuji**
Changes in this PR:

- tests/rt_gnu.conf: turn off IPD tests, no longer working on Cheyenne with GNU compiler (segmentation fault)
- update of jet modulefile and correction of cmake name from jet to jet.intel (to match gnumake)

No need to run regression tests on any of the platforms, since the changes are confined to files that are not used in any of these tests.

Todo in future PRs:
- re-enable regression testing on jet.intel, possibly update NCEPlibs installation
- clean up various `rt*.conf` regression test configs

### CODE REVIEW

**Comment by climbfuji** [comment]
Testing on Cheyenne with both Intel and GNU:
- create new baselines using `rt.sh -l rt_stampede_cheyenne.conf -c -e` (for Intel) or `rt.sh -l rt_gnu.conf -c -e` (for GNU)
- rsync `NEW_BASELINE` directories to `RTPWD`
- verify against baselines using `rt.sh -l rt_stampede_cheyenne.conf -e` (for Intel) or `rt.sh -l rt_gnu.conf -e` (for GNU)

All tests pass.

[rt_cheyenne_intel_create.log](https://github.com/ufs-community/ufs-weather-model/files/4717942/rt_cheyenne_intel_create.log)
[rt_cheyenne_intel_verify.log](https://github.com/ufs-community/ufs-weather-model/files/4717943/rt_cheyenne_intel_verify.log)
[rt_cheyenne_gnu_create.log](https://github.com/ufs-community/ufs-weather-model/files/4717944/rt_cheyenne_gnu_create.log)
[rt_cheyenne_gnu_verify.log](https://github.com/ufs-community/ufs-weather-model/files/4717945/rt_cheyenne_gnu_verify.log)

**Comment by climbfuji** [comment]
Testing on Jet with Intel: I successfully compiled the code using the following two commands (all CCPP suites):

cmake (PROD mode)
```
./compile_cmake.sh $PWD/.. jet.intel 'CCPP=Y' '' NO NO 2>&1 | tee compile.log
```
gnumake (DEBUG mode)
```
 ./compile.sh $PWD/../FV3 jet.intel 'CCPP=Y DEBUG=Y' '' NO NO 2>&1 | tee compile.log
```

**Review by DusanJovic-NOAA — APPROVED** [review]

File: modulefiles/jet.intel/fv3 [hunk 1]
```diff
@@ -20,7 +20,7 @@ module load impi/2018.4.274
 module load hdf5/1.10.4
 module load netcdf/4.6.1
 
-module use -a /lfs3/projects/hfv3gfs/GMTB/modulefiles/intel-18.0.5.274
+module use -a /lfs4/HFIP/hfv3gfs/software/modulefiles/intel-18.0.5.274/impi-2018.4.274
 
 ##
 ### NCEP libraries (download and build manually for the time being; https://github.com/NCAR/NCEPlibs)
```
Comment (junwang-noaa) [inline modulefiles/jet.intel/fv3:26]: Since NCEPlibs team is rebuilding nceplibs, I expect we will change the nceplibs modules files later?

**Review by junwang-noaa — APPROVED** [review]

File: modulefiles/jet.intel/fv3 (hunk 1 above)
Comment (climbfuji) [reply modulefiles/jet.intel/fv3:26]: Yes, as noted above in the description of the PR. For now, the SAR real-time parallels can proceed with this version (same as they used until now).

**Pull request merged**